### PR TITLE
feat(editorconfig): add editorconfig snippets

### PR DESCRIPTION
**Pull request by pbnj**
Adds snippets for `.editorconfig` files

### CODE REVIEW

**Comment by rafamadriz** [comment]
You would need to add the path to package.json, otherwise snippets aren't automatically loaded

**Comment by pbnj** [comment]
@rafamadriz - Done. Thank you.

**Review by OkelleyDevelopment — APPROVED** [review]

File: snippets/editorconfig.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "root": {
```
Comment (OkelleyDevelopment) [inline snippets/editorconfig.json:2]: Before I merge, can you (@pbnj) confirm if this "root" snippet is supposed to be the "boilerplate" that is added to the `.editorconfig` file?

Comment (pbnj) [reply snippets/editorconfig.json:2]: That is correct. The `root` stanza is a boilerplate for an `.editorconfig` file

Comment (OkelleyDevelopment) [reply snippets/editorconfig.json:2]: Awesome thanks!

**Pull request merged**